src/components/expanded-state/swap-settings/RoutePicker.js

```diff
@@ -0,0 +1,101 @@
+import lang from 'i18n-js';
+import React, { useCallback } from 'react';
+import RainbowExchange from '../../../assets/exchanges/Both.png';
+import OneInchExchange from '../../../assets/exchanges/oneinch.png';
+import ZeroXExchange from '../../../assets/exchanges/zerox.png';
+import { ContextMenuButton } from '../../context-menu';
+import { Column, Columns, Inline, Text } from '@rainbow-me/design-system';
+
+import { ImgixImage } from '@rainbow-me/images';
+import { SwapRoute } from '@rainbow-me/redux/swap';
+import { showActionSheetWithOptions } from '@rainbow-me/utils';
+
+const routeMenuItems = () => {
+  return Object.values(SwapRoute).map(route => ({
+    actionKey: route,
+    actionTitle: Object.keys(SwapRoute).find(key => SwapRoute[key] === route),
+    icon: {
+      iconType: 'ASSET',
+      iconValue: `${route}`,
+    },
+  }));
+};
+
+const androidRouteMenuItems = () => {
+  return Object.values(SwapRoute);
+};
+
+export default function SwapSettingsState({ onSelect, swapRoute }) {
+  const imageSource = useMemo(() => {
+    let source = null;
+    switch (swapRoute) {
+      case SwapRoute['1inch']:
```

Comment:
We should use  `Source.Aggregator0x`  and  `Source.Aggregotor1inch`

Reply:
fixed